docs/atomic-uninstall.1.md

```diff
@@ -49,5 +50,8 @@ Any additional arguments will be appended to the command.
 **-n** **--name**=""
    If name is specified `atomic uninstall` will uninstall the named container from the system, otherwise it will uninstall the container images.
 
+**--system**
+   Uninstall a system container.
```

Comment:
Is there anyways we can figure out that this is a system container?  Should we leave a bit of information around to tell us that the container is a system container so atomic uninstall will just do the right thing.  If I am running a system container and accidently do not include the --system flag, what happens?